packages/lib/src/components/SecuredFields/SecuredFieldsInput/SecuredFieldsInput.tsx

```diff
@@ -0,0 +1,97 @@
+import { h } from 'preact';
+import { useState, useEffect, useRef, useMemo } from 'preact/hooks';
+import Language from '../../../language/Language';
+import SecuredFieldsProvider, { SFPState } from '../../internal/SecuredFields/SecuredFieldsProvider';
+import { BinLookupResponse } from '../../Card/types';
+import SFExtensions from '../../internal/SecuredFields/binLookup/extensions';
+
+interface SecuredFieldsProps {
+    onChange: (data) => void;
+    i18n: Language;
+}
+
+const defaultProps = {
+    onChange: () => {},
+    onError: () => {}
+};
+
+function SecuredFields(props: SecuredFieldsProps) {
```

Comment:
Should this be called `SecuredFieldsInput`?

Reply:
I'm fed up of too many things being called SecuredFields in too many directories of the same name.
So I'm following the pattern of the Card comp: `Card` & `CardInput`

Reply:
Misunderstood. Have changed the internal name to SecuredFieldsInput